backend/build.gradle

```diff
@@ -219,8 +219,3 @@ test {
     }
 }
 
-sourceSets {
-  main {
-    java.srcDirs += "build/generated/sources/annotationProcessor/java/main"
```

Comment:
I got rid of this because it was having weird side effects and causing the app to not always build properly ¯\\(°_o)/¯